src/key.rs

```diff
@@ -291,6 +290,18 @@ impl PublicKey {
         ret
     }
 
+    /// Writes the key as a byte-encoded pair of values. In compressed form
+    /// the y-coordinate is represented by only a single bit, as x determines
+    /// it up to one bit.
+    pub fn write_compressed<W: io::Write>(&self, writer: &mut W) -> io::Result<()>{
```

Comment:
Probably should be called `write` because we also have `serialize` and `serialize_uncompressed` (and `PUBLIC_KEY_SIZE` and `UNCOMPRESSED_PUBLIC_KEY_SIZE`).